stage0/src/sev.rs

```diff
@@ -34,10 +33,21 @@ use x86_64::{
     PhysAddr, VirtAddr,
 };
 
-#[link_section = ".boot.ghcb"]
-static mut GHCB: MaybeUninit<Ghcb> = MaybeUninit::uninit();
+static GHCB_WRAPPER: OnceCell<Spinlock<GhcbProtocol<'static, SevGhcb>>> = OnceCell::new();
 
-static GHCB_WRAPPER: OnceCell<Spinlock<GhcbProtocol<'static, Ghcb>>> = OnceCell::new();
+#[repr(transparent)]
```

Comment:
Perhaps add a `why` comment here, since `Ghcb` is wrapping another `Ghcb` from another crate. Why is this wrapping needed?

Reply:
I got rid of it; it was a remnant of me trying to invent something more Rust-y around the GHCB before giving up.